index.html

```
@@ -0,0 +1,672 @@
+               </div>
+            </div>
+         </section>
+      </main>
+      <!-- Image Modal -->
+      <div class="modal" id="image-modal" onclick="closeModal()">
+         <button class="modal-close" onclick="closeModal()">
+            <svg class="w-5 h-5" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+               <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12"/>
+            </svg>
+         </button>
+         <img id="modal-img" src="" alt="Full size">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)